core/utils/toolbox.js

```diff
@@ -387,7 +386,7 @@ const addAttributes = function(node, obj) {
 const parseToolboxTree = function(toolboxDef) {
   if (toolboxDef) {
     if (typeof toolboxDef != 'string') {
-      if (Blockly.utils.userAgent.IE && toolboxDef.outerHTML) {
+      if (IE && toolboxDef.outerHTML) {
```

Comment:
This (and the corresponding change to the `require` above) is certainly mechanically correct, but I don't think the result is very readable.  Probably the API of the `Blocly.utils.userAgent` module could be improved, but for now I suggest importing all of `useragent` and then writing `useragent.IE`.  That requires no changes outside this file but would help reduce reader confusion.